### ISSUE TITLE
Bug with multi-word camel case variables

### ISSUE THREAD

**Issue by dylan-chong** [chiedo/vim-case-convert#3]
Hi! Thanks for writing this plugin, it's very handy :)

There is a bug where if you select

      helloWorldBoo

and run `:'<,'>CamelToSnake`, then `y` will only convert the first word and we'll be left with

      hello_worldBoo

**Comment by Arnie97** [comment]
Nope, I think this is the expected behavior. In your scenario, choose `a` rather than `y`, or just use the "bang" version `:CamelToSnake!`.

**Comment by dylan-chong** [comment]
Nope. They both behave the same as the problem in the issue :(
